src/mmw/js/src/modeling/models.js

```diff
@@ -103,8 +136,8 @@ var ProjectModel = Backbone.Model.extend({
 
 var ModificationModel = coreModels.GeoModel.extend({
     defaults: _.extend({
-            name: '',
-            type: ''
+            'name': '',
+            'type': ''
```

Comment:
I feel like the only attribute keys that should be quoted are ones that contain special chars or symbols. Since model attributes should more or less conform to variable naming conventions, I would rather not quote these.